src/main/resources/application-local.yml

```diff
@@ -0,0 +1,17 @@
+spring:
```

Comment:
If this file is strictly for your local environment please keep it away from of repository files